ccd-definition/AuthorisationCaseType/AuthorisationCaseType-nonprod-shuttered.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
```

Comment:
since nonprod files will always be unshuttered so no need to do anything to the file AuthorisationCaseType-nonprod.json

Reply:
thanks, I've removed the -shuttered prefix from this file.